validator/distro_validator.go

```diff
@@ -549,3 +550,41 @@ func ensureHasValidVirtualWorkstationSettings(ctx context.Context, d *distro.Dis
 	}
 	return errs
 }
+
+func validateAliases(ctx context.Context, d *distro.Distro, allDistroAliases []string) ValidationErrors {
+	var validationErrs ValidationErrors
+	// Parent and container distros do not support aliases.
+	if d.ContainerPool != "" || d.Provider == evergreen.ProviderNameDocker {
+		validationErrs = append(validationErrs, ensureNoAliases(d, allDistroAliases)...)
+	}
+	if len(d.Aliases) > 0 {
+		validationErrs = append(validationErrs, ensureValidAliases(d)...)
+	}
+	return validationErrs
+}
+
+// ValidateDistroSection validates that the incoming changes to a given section are valid.
+func ValidateDistroSection(ctx context.Context, originalDistro *distro.Distro, changes *distro.Distro, section distro.DistroSettingsSection, s *evergreen.Settings) error {
```

Comment:
[nit] I would probably grab settings from the Evergreen environment in this function directly, your call though!